test/oauth-server/config.yaml

```diff
@@ -0,0 +1,14 @@
+log:
+  debug: false
+apis:
+  http:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+oauthSigner:
+  idTokenKeyFile: "/secrets/private/idToken.key"
+  accessTokenKeyFile: "/secrets/private/accessToken.key"
+  domain: ""
```

Comment:
What is this domain used for?

Reply:
It used as:

- issuer for idToken and accessToken
- audience for accessToken

Reply:
@Trishia please document what exactly is it used for.